sentry-android-core/src/main/res/layout/sentry_dialog_user_feedback.xml

```diff
@@ -0,0 +1,112 @@
+        android:layout_width="32dp"
+        android:layout_height="32dp"
+        android:layout_alignTop="@+id/sentry_dialog_user_feedback_title"
+        android:layout_alignBottom="@+id/sentry_dialog_user_feedback_title"
+        android:layout_alignParentEnd="true"
+        android:contentDescription="Logo of the brand"
+        android:tint="?android:attr/colorForeground"
+        android:src="@drawable/sentry_logo_dark"/>
+
+    <TextView
+        android:id="@+id/sentry_dialog_user_feedback_txt_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Name"
+        android:layout_marginTop="4dp"
+        android:textStyle="bold"
+        android:layout_below="@id/sentry_dialog_user_feedback_title" />
+
+    <EditText
+        android:id="@+id/sentry_dialog_user_feedback_edt_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="Your Name"
+        android:inputType="textPersonName"
+        android:background="@drawable/edit_text_border"
+        android:paddingHorizontal="8dp"
+        android:layout_below="@id/sentry_dialog_user_feedback_txt_name" />
+
+    <TextView
+        android:id="@+id/sentry_dialog_user_feedback_txt_email"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Email"
+        android:layout_marginTop="8dp"
+        android:textStyle="bold"
+        android:layout_below="@id/sentry_dialog_user_feedback_edt_name" />
+
+    <EditText
+        android:id="@+id/sentry_dialog_user_feedback_edt_email"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="your.email@example.org"
+        android:inputType="textEmailAddress"
+        android:background="@drawable/edit_text_border"
+        android:paddingHorizontal="8dp"
+        android:layout_below="@id/sentry_dialog_user_feedback_txt_email" />
+
+    <TextView
+        android:id="@+id/sentry_dialog_user_feedback_txt_description"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Description (Required)"
+        android:layout_marginTop="8dp"
+        android:textStyle="bold"
+        android:layout_below="@id/sentry_dialog_user_feedback_edt_email" />
+
+    <EditText
+        android:id="@+id/sentry_dialog_user_feedback_edt_description"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:lines="6"
+        android:inputType="textMultiLine"
+        android:gravity="top|start"
+        android:hint="What's the bug? What did you expect?"
+        android:background="@drawable/edit_text_border"
+        android:paddingHorizontal="8dp"
+        android:layout_below="@id/sentry_dialog_user_feedback_txt_description" />
+
+    <Button
+        android:id="@+id/sentry_dialog_user_feedback_btn_send"
```

Comment:
Could we have the buttons horizontally, side by side? At least that would be more Android-ish.
See https://developer.android.com/develop/ui/views/components/dialogs
![image](https://github.com/user-attachments/assets/88254a83-2764-459a-85aa-1d798d2c9e75)